.github/workflows/int-and-connected-test-run.yml

```diff
@@ -1,103 +1,42 @@
 name: Unit, Smoke, Connected and Integration tests
-env:
-  K8_CLUSTER: 'integration-master'
```

Comment:
Reviewer note: I think it's better to pass the minimum number of required environment variables to each step to see *exactly* what they need to run

Reply:
I agree with this shift as a general rule.  What would you think about making an exception for this variable?  It feels like a general setting that we want for all tests.

```
  # This will make test logs not use JSON
  TDR_LOG_APPENDER: 'Console-Standard'
```